extras/rya.streams/api/src/main/java/org/apache/rya/streams/api/queries/QueryRepository.java

```diff
@@ -25,14 +25,16 @@
 import org.apache.rya.streams.api.entity.StreamsQuery;
 import org.apache.rya.streams.api.exception.RyaStreamsException;
 
+import com.google.common.util.concurrent.Service;
+
 import edu.umd.cs.findbugs.annotations.DefaultAnnotation;
 import edu.umd.cs.findbugs.annotations.NonNull;
 
 /**
  * Repository for adding, deleting, and listing active queries in Rya Streams.
  */
 @DefaultAnnotation(NonNull.class)
-public interface QueryRepository extends AutoCloseable {
+public interface QueryRepository extends AutoCloseable, Service {
```

Comment:
Since this is a Service now, it probably should no longer be AutoCloseable.

Reply:
ok